Sources/TokamakCore/Modifiers/FlexFrameLayout.swift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public struct _FlexFrameLayout: ViewModifier {
+  public let minWidth: CGFloat?
+  public let idealWidth: CGFloat?
+  public let maxWidth: CGFloat?
+  public let minHeight: CGFloat?
+  public let idealHeight: CGFloat?
+  public let maxHeight: CGFloat?
+  public let alignment: Alignment
+
+  init(minWidth: CGFloat? = nil, idealWidth: CGFloat? = nil, maxWidth: CGFloat? = nil, minHeight: CGFloat? = nil, idealHeight: CGFloat? = nil, maxHeight: CGFloat? = nil, alignment: Alignment) {
```

Comment:
**nitpick:** Please avoid long lines, if you set up `pre-commit` as described in [the "Coding Style" section](https://github.com/swiftwasm/Tokamak#coding-style) it will prevent you from committing these as it will run `swiftformat` and `swiftlint` before you commit.

Reply:
I setup the pre-commit and fixed the long lines.

Reply:
Awesome, thank you 🙏